app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,40 @@
+class PostsController < ApplicationController
+    def index
+        @posts = Post.all
+    end
+
+    def new
+        @post = Post.new
+    end
+
+    def show
+        @post = Post.find(params[:id])
+    end
+
+    def create
+        @post = Post.create! post_params
+        redirect_to posts_path
+    end
+
+    def edit
+        @post = Post.find(params[:id])
+    end
+
+    def update
+        @post = Post.find(params[:id])
+        @post = Post.update(post_params)
+        redirect_to posts_path(@post)
+    end
+
+    def destroy
+        @post = Post.find(params[:id])
+        @post.destroy
+        redirect_to posts_path
+    end
+
+
+    private
+    def post_params
+        params.require(:post).permit(:title, :content, :created_at, :updated_at)
+    end
```

Comment:
Similar to my note in the comments controller: do you need `created_at` and `updated_at` in here?